picard/ui/caa_types_selector.py

```diff
@@ -325,7 +325,7 @@ def included(self):
     def excluded(self):
         return list(self.list_exclude.all_items_data()) or ['none']
 
-    def clear_focus(self, lists):
+    def clear_focus(self, lists, *argc):
```

Comment:
If it's one parameter, why using `*argc` ? (and why `*argc` instead of more common`*args`)

I would go for an even more explicit variable name to match https://doc.qt.io/qt-6/qabstractitemview.html#clicked

```python
def clear_focus(self, lists, qmodelindex):
```

Reply:
argc is really just a typo, it should be args of course.

But I chose `*args`  because we really don't care about any extra arguments here, and the values passed don't have any further meaning for the function. I first had it as `clear_focus(self, lists, index)` , but it seemed confusing why the `clear_focus` function would have an index parameter and why it needs it. It only makes sense if you know that this is a "clicked" slot.

We could also maybe just rename this method to e.g. `def _on_list_clicked(self, lists, index):` to make things clearer

Reply:
I did the renaming in 4f15d262e9b937139e398784419533213de01274

Reply:
> We could also maybe just rename this method to e.g. `def _on_list_clicked(self, lists, index):` to make things clearer

The problem I have with `index` is that's not very clear that's a QModelIndex object as we use it a lot for integer indexes, but no strong opinion about this, especially since in this case this parameter is unused.
But I agree with the function's name, using `_on` (or `on_`) prefix makes it clearer that's a signal handler.

Maybe:

```python
def def _on_list_clicked(self, lists, *args):
```